packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -68,28 +72,43 @@ export class AtomicSearchBox {
    */
   @Prop() public numberOfQueries = 8;
 
+  /**
+   * Defining this options makes the search box standalone.
+   *
+   * The default Url the user should be redirected to, when a query is submitted.
+   * If a query pipeline redirect is triggered, it will redirect to that Url instead.
+   */
+  @Prop() public redirectionUrl?: string;
```

Comment:
I decided to adapt the current search box, it doesn't add much complexity since the controllers have the same interface. The complexity would be greater in copying & sharing code in between 2 component. 

For a user of the library, the setup is basically the same for both and the search box behaves the same, setting a simple prop handles everything in an elegant manner.

Reply:
I guess that's fine ... 🤔   The internal complexity is not really what interest me though.

My main concern is if someone searches for "how to create a standalone search box atomic", will they discover/stumble on that prop on their own ?

I think it's a fair assumption to say that they would look for properties in the search box component and  ultimately discover this one relatively easily.

If we had a unique component for this, however, I think it might be a bit more easily discoverable overall.

If you scanned the list of all components in our doc, and saw two different search box, it might be more easy to understand that there's two different "type" of search box that behave a bit differently.

However with all that said, I think it's fine to leave it like you have done it here.

Reply:
I'd add a usage page with more details on how to set it up too